manifests/t/tazzben/Assessment-Disaggregation/1.5.3/tazzben.Assessment-Disaggregation.installer.yaml

```
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: tazzben.Assessment-Disaggregation
+PackageVersion: 1.5.3
```

Uhm, for some reason this installs 1.5.2 and the setup is called 1.5.2, however the hash is different from the 1.5.2 release so kinda conflicted about this one
```suggestion
PackageVersion: 1.5.2
```